arrebol/worker/manager/manager.go

```diff
@@ -0,0 +1,22 @@
+package manager
+
```

Comment:
is this the name @raonismaneoto suggested in his design? what is a manager?

Reply:
I updated it. This entity must be an instance of `WorkerScheduler` from diagram.

Reply:
i try to avoiding using *Manager to name things ... but, ok